opnode/rollup/driver/state.go

```diff
@@ -312,53 +302,71 @@ func (s *state) handleEpoch(ctx context.Context) (bool, error) {
 func (s *state) loop() {
 	defer s.wg.Done()
 	s.log.Info("State loop started")
+
```

Comment:
The `loop` function seems to have changed quite a bit on main, so probably this will need a rebase/conflict resolution.